%YAML:1.0
---
captureNum: 1
cameraMatrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [561.724858, 0.000000, 647.253365, 0.000000, 561.047084, 335.914659, 0.000000, 0.000000, 1.000000]
distCoeffs: !!opencv-matrix
   rows: 5
   cols: 1
   dt: d
   data: [-0.000848, -0.036099, -0.001363, 0.007765, 0.000000]
H: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [-5.48909425e-01, -1.68627027e+00,  9.67436661e+02, -5.40948616e-02, -4.39986068e-02, -2.35435150e+02, -4.05035266e-05, -2.65265882e-03,  1.00000000e+00]

focus: 2.6
height: 720
width: 1280
dist_each_pix_x: 4.478815274010935
dist_each_pix_y: 4.337648882269004
each_pix_in_sensor: 0.0047339785856874966
